app/views/posts/show.html.erb

```diff
@@ -1 +1,20 @@
-<p> POSTS SHOW </p>
\ No newline at end of file
+<article class="w-100 bord p-2 rounded-top mb-4 my-5 shadow">
+  <section class="d-flex justify-content-between postsBy">
+    <div>
+      <h5>Post # <%=@post.id %> by <%= @post.user.name %></h5>
+    </div>
+    <div class="align-self-end">
+      <p>Comments: <%=@post.comments.count%> Likes: <%=@post.likes.count%></p>
+    </div>
+  </section>
+  <section class="postsText my-3">
+    <p><%= @post.text%></p>
+  </section>
+
+  <section class="bord p-2 shadow">
+  <% @post.comments.each do |comment| %>
+    <p><%= comment.user.name %> : <%= comment.text %> 
+  <% end %>
+  </section>
```

Comment:
- [ ] When showing a post you should have only 5recent comments but currently you are rendering all comments please work on that